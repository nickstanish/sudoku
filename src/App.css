html {
  box-sizing: border-box;
}

*, *:before, *:after {
  box-sizing: inherit;
}

.App {
  text-align: center;
}


.App-link {
  color: #61dafb;
}

.feather {
  width: 24px;
  height: 24px;
  stroke: currentColor;
  stroke-width: 2;
  stroke-linecap: round;
  stroke-linejoin: round;
  fill: none;
}

.touch-only {
  display: none;
}

.is-touch .touch-only {
  display: initial;
}