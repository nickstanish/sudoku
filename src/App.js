import React from 'react';
import './App.css';
import GameContainer from './GameContainer';

function App() {
  return (
    <div className="App">
      <GameContainer />
    </div>
  );
}

export default App;
