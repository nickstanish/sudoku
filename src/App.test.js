import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

jest.mock('./GameContainer', () => () => 'GameContainer');

it('renders', () => {
  const { container } = render(<App />);
  expect(container).toMatchSnapshot();
});
