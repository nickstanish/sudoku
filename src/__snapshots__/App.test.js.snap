// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div>
  <div
    class="App"
  >
    GameContainer
  </div>
</div>
`;
